### PR TITLE
[SQL]: Fix #3365 #3367 fix codes table

### LINKED ISSUE #3365: SQL Mode issues while installing OpenEMR
### Describe the bug

After changing the modes in the Installer.class.php and try running installation via GUI following error occurs:
```ERROR. unable to execute SQL: 'INSERT INTO `codes` (`id`, `code_text`, `code_text_short`, `code`, `code_type`, `modifier`, `units`, `fee`, `superbill`, `related_code`, `taxrates`, `cyp_factor`, `active`)VALUES(NULL, "diphtheria, tetanus toxoids and pertussis vaccine", "DTP", 1, 100, '', 0, 0, '', '', '', '', 1),(NULL, "poliovirus vaccine, live, oral", "OPV", 2, 100, '', 0, 0, '', '', '', '', 1),(NULL, "measles, mumps and rubella virus vaccine", "MMR", 3, 100, '', 0, 0, '', '', '', '', 1),(NULL, "measles and rubella virus vaccine", "M/R", 4, 100, '', 0, 0, '', '', '', '', 1),(NULL, "measles virus vaccine", "measles", 5, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rubella virus vaccine", "rubella", 6, 100, '', 0, 0, '', '', '', '', 1),(NULL, "mumps virus vaccine", "mumps", 7, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis B vaccine, pediatric or pediatric/adolescent dosage", "Hep B, adolescent or pediatric", 8, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus and diphtheria toxoids, adsorbed, for adult use", "Td (adult), adsorbed", 9, 100, '', 0, 0, '', '', '', '', 1),(NULL, "poliovirus vaccine, inactivated", "IPV", 10, 100, '', 0, 0, '', '', '', '', 1),(NULL, "pertussis vaccine", "pertussis", 11, 100, '', 0, 0, '', '', '', '', 1),(NULL, "diphtheria antitoxin", "diphtheria antitoxin", 12, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus immune globulin", "TIG", 13, 100, '', 0, 0, '', '', '', '', 1),(NULL, "immune globulin, unspecified formulation", "IG, unspecified formulation", 14, 100, '', 0, 0, '', '', '', '', 1),(NULL, "influenza virus vaccine, split virus (incl. purified surface antigen)-retired CODE", "influenza, split (incl. purified surface antigen)", 15, 100, '', 0, 0, '', '', '', '', 1),(NULL, "influenza virus vaccine, whole virus", "influenza, whole", 16, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Haemophilus influenzae type b vaccine, conjugate unspecified formulation", "Hib, unspecified formulation", 17, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rabies vaccine, for intramuscular injection", "rabies, intramuscular injection", 18, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Bacillus Calmette-Guerin vaccine", "BCG", 19, 100, '', 0, 0, '', '', '', '', 1),(NULL, "diphtheria, tetanus toxoids and acellular pertussis vaccine", "DTaP", 20, 100, '', 0, 0, '', '', '', '', 1),(NULL, "varicella virus vaccine", "varicella", 21, 100, '', 0, 0, '', '', '', '', 1),(NULL, "DTP-Haemophilus influenzae type b conjugate vaccine", "DTP-Hib", 22, 100, '', 0, 0, '', '', '', '', 1),(NULL, "plague vaccine", "plague", 23, 100, '', 0, 0, '', '', '', '', 1),(NULL, "anthrax vaccine", "anthrax", 24, 100, '', 0, 0, '', '', '', '', 1),(NULL, "typhoid vaccine, live, oral", "typhoid, oral", 25, 100, '', 0, 0, '', '', '', '', 1),(NULL, "cholera vaccine", "cholera", 26, 100, '', 0, 0, '', '', '', '', 1),(NULL, "botulinum antitoxin", "botulinum antitoxin", 27, 100, '', 0, 0, '', '', '', '', 1),(NULL, "diphtheria and tetanus toxoids, adsorbed for pediatric use", "DT (pediatric)", 28, 100, '', 0, 0, '', '', '', '', 1),(NULL, "cytomegalovirus immune globulin, intravenous", "CMVIG", 29, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis B immune globulin", "HBIG", 30, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis A vaccine, pediatric dosage, unspecified formulation", "Hep A, pediatric, unspecified formulation", 31, 100, '', 0, 0, '', '', '', '', 1),(NULL, "meningococcal polysaccharide vaccine (MPSV4)", "meningococcal MPSV4", 32, 100, '', 0, 0, '', '', '', '', 1),(NULL, "pneumococcal polysaccharide vaccine, 23 valent", "pneumococcal polysaccharide PPV23", 33, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rabies immune globulin", "RIG", 34, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus toxoid, adsorbed", "tetanus toxoid, adsorbed", 35, 100, '', 0, 0, '', '', '', '', 1),(NULL, "varicella zoster immune globulin", "VZIG", 36, 100, '', 0, 0, '', '', '', '', 1),(NULL, "yellow fever vaccine", "yellow fever", 37, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rubella and mumps virus vaccine", "rubella/mumps", 38, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Japanese Encephalitis Vaccine SC", "Japanese encephalitis SC", 39, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rabies vaccine, for intradermal injection", "rabies, intradermal injection", 40, 100, '', 0, 0, '', '', '', '', 1),(NULL, "typhoid vaccine, parenteral, other than acetone-killed, dried", "typhoid, parenteral", 41, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis B vaccine, adolescent/high risk infant dosage", "Hep B, adolescent/high risk infant", 42, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis B vaccine, adult dosage", "Hep B, adult", 43, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis B vaccine, dialysis patient dosage", "Hep B, dialysis", 44, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis B vaccine, unspecified formulation", "Hep B, unspecified formulation", 45, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Haemophilus influenzae type b vaccine, PRP-D conjugate", "Hib (PRP-D)", 46, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Haemophilus influenzae type b vaccine, HbOC conjugate", "Hib (HbOC)", 47, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Haemophilus influenzae type b vaccine, PRP-T conjugate", "Hib (PRP-T)", 48, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Haemophilus influenzae type b vaccine, PRP-OMP conjugate", "Hib (PRP-OMP)", 49, 100, '', 0, 0, '', '', '', '', 1),(NULL, "DTaP-Haemophilus influenzae type b conjugate vaccine", "DTaP-Hib", 50, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Haemophilus influenzae type b conjugate and Hepatitis B vaccine", "Hib-Hep B", 51, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis A vaccine, adult dosage", "Hep A, adult", 52, 100, '', 0, 0, '', '', '', '', 1),(NULL, "typhoid vaccine, parenteral, acetone-killed, dried (U.S. military)", "typhoid, parenteral, AKD (U.S. military)", 53, 100, '', 0, 0, '', '', '', '', 1),(NULL, "adenovirus vaccine, type 4, live, oral", "adenovirus, type 4", 54, 100, '', 0, 0, '', '', '', '', 1),(NULL, "adenovirus vaccine, type 7, live, oral", "adenovirus, type 7", 55, 100, '', 0, 0, '', '', '', '', 1),(NULL, "dengue fever vaccine", "dengue fever", 56, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hantavirus vaccine", "hantavirus", 57, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis C vaccine", "Hep C", 58, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis E vaccine", "Hep E", 59, 100, '', 0, 0, '', '', '', '', 1),(NULL, "herpes simplex virus, type 2 vaccine", "herpes simplex 2", 60, 100, '', 0, 0, '', '', '', '', 1),(NULL, "human immunodeficiency virus vaccine", "HIV", 61, 100, '', 0, 0, '', '', '', '', 1),(NULL, "human papilloma virus vaccine, quadrivalent", "HPV, quadrivalent", 62, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Junin virus vaccine", "Junin virus", 63, 100, '', 0, 0, '', '', '', '', 1),(NULL, "leishmaniasis vaccine", "leishmaniasis", 64, 100, '', 0, 0, '', '', '', '', 1),(NULL, "leprosy vaccine", "leprosy", 65, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Lyme disease vaccine", "Lyme disease", 66, 100, '', 0, 0, '', '', '', '', 1),(NULL, "malaria vaccine", "malaria", 67, 100, '', 0, 0, '', '', '', '', 1),(NULL, "melanoma vaccine", "melanoma", 68, 100, '', 0, 0, '', '', '', '', 1),(NULL, "parainfluenza-3 virus vaccine", "parainfluenza-3", 69, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Q fever vaccine", "Q fever", 70, 100, '', 0, 0, '', '', '', '', 1),(NULL, "respiratory syncytial virus immune globulin, intravenous", "RSV-IGIV", 71, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rheumatic fever vaccine", "rheumatic fever", 72, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Rift Valley fever vaccine", "Rift Valley fever", 73, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rotavirus, live, tetravalent vaccine", "rotavirus, tetravalent", 74, 100, '', 0, 0, '', '', '', '', 1),(NULL, "vaccinia (smallpox) vaccine", "vaccinia (smallpox)", 75, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Staphylococcus bacteriophage lysate", "Staphylococcus bacterio lysate", 76, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tick-borne encephalitis vaccine", "tick-borne encephalitis", 77, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tularemia vaccine", "tularemia vaccine", 78, 100, '', 0, 0, '', '', '', '', 1),(NULL, "vaccinia immune globulin", "vaccinia immune globulin", 79, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Venezuelan equine encephalitis, live, attenuated", "VEE, live", 80, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Venezuelan equine encephalitis, inactivated", "VEE, inactivated", 81, 100, '', 0, 0, '', '', '', '', 1),(NULL, "adenovirus vaccine, unspecified formulation", "adenovirus, unspecified formulation", 82, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis A vaccine, pediatric/adolescent dosage, 2 dose schedule", "Hep A, ped/adol, 2 dose", 83, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis A vaccine, pediatric/adolescent dosage, 3 dose schedule", "Hep A, ped/adol, 3 dose", 84, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis A vaccine, unspecified formulation", "Hep A, unspecified formulation", 85, 100, '', 0, 0, '', '', '', '', 1),(NULL, "immune globulin, intramuscular", "IG", 86, 100, '', 0, 0, '', '', '', '', 1),(NULL, "immune globulin, intravenous", "IGIV", 87, 100, '', 0, 0, '', '', '', '', 1),(NULL, "influenza virus vaccine, unspecified formulation", "influenza, unspecified formulation", 88, 100, '', 0, 0, '', '', '', '', 1),(NULL, "poliovirus vaccine, unspecified formulation", "polio, unspecified formulation", 89, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rabies vaccine, unspecified formulation", "rabies, unspecified formulation", 90, 100, '', 0, 0, '', '', '', '', 1),(NULL, "typhoid vaccine, unspecified formulation", "typhoid, unspecified formulation", 91, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Venezuelan equine encephalitis vaccine, unspecified formulation", "VEE, unspecified formulation", 92, 100, '', 0, 0, '', '', '', '', 1),(NULL, "respiratory syncytial virus monoclonal antibody (palivizumab), intramuscular", "RSV-MAb", 93, 100, '', 0, 0, '', '', '', '', 1),(NULL, "measles, mumps, rubella, and varicella virus vaccine", "MMRV", 94, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tuberculin skin test; old tuberculin, multipuncture device", "TST-OT tine test", 95, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tuberculin skin test; purified protein derivative solution, intradermal", "TST-PPD intradermal", 96, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tuberculin skin test; purified protein derivative, multipuncture device", "TST-PPD tine test", 97, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tuberculin skin test; unspecified formulation", "TST, unspecified formulation", 98, 100, '', 0, 0, '', '', '', '', 1),(NULL, "RESERVED - do not use", "RESERVED - do not use", 99, 100, '', 0, 0, '', '', '', '', 1),(NULL, "pneumococcal conjugate vaccine, 7 valent", "pneumococcal conjugate PCV 7", 100, 100, '', 0, 0, '', '', '', '', 1),(NULL, "typhoid Vi capsular polysaccharide vaccine", "typhoid, ViCPs", 101, 100, '', 0, 0, '', '', '', '', 1),(NULL, "DTP- Haemophilus influenzae type b conjugate and hepatitis b vaccine", "DTP-Hib-Hep B", 102, 100, '', 0, 0, '', '', '', '', 1),(NULL, "meningococcal C conjugate vaccine", "meningococcal C conjugate", 103, 100, '', 0, 0, '', '', '', '', 1),(NULL, "hepatitis A and hepatitis B vaccine", "Hep A-Hep B", 104, 100, '', 0, 0, '', '', '', '', 1),(NULL, "vaccinia (smallpox) vaccine, diluted", "vaccinia (smallpox) diluted", 105, 100, '', 0, 0, '', '', '', '', 1),(NULL, "diphtheria, tetanus toxoids and acellular pertussis vaccine, 5 pertussis antigens", "DTaP, 5 pertussis antigens", 106, 100, '', 0, 0, '', '', '', '', 1),(NULL, "diphtheria, tetanus toxoids and acellular pertussis vaccine, unspecified formulation", "DTaP, unspecified formulation", 107, 100, '', 0, 0, '', '', '', '', 1),(NULL, "meningococcal vaccine, unspecified formulation", "meningococcal, unspecified formulation", 108, 100, '', 0, 0, '', '', '', '', 1),(NULL, "pneumococcal vaccine, unspecified formulation", "pneumococcal, unspecified formulation", 109, 100, '', 0, 0, '', '', '', '', 1),(NULL, "DTaP-hepatitis B and poliovirus vaccine", "DTaP-Hep B-IPV", 110, 100, '', 0, 0, '', '', '', '', 1),(NULL, "influenza virus vaccine, live, attenuated, for intranasal use", "influenza, live, intranasal", 111, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus toxoid, unspecified formulation", "tetanus toxoid, unspecified formulation", 112, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus and diphtheria toxoids, adsorbed, preservative free, for adult use", "Td (adult) preservative free", 113, 100, '', 0, 0, '', '', '', '', 1),(NULL, "meningococcal polysaccharide (groups A, C, Y and W-135) diphtheria toxoid conjugate vaccine (MCV4P)", "meningococcal MCV4P", 114, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus toxoid, reduced diphtheria toxoid, and acellular pertussis vaccine, adsorbed", "Tdap", 115, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rotavirus, live, pentavalent vaccine", "rotavirus, pentavalent", 116, 100, '', 0, 0, '', '', '', '', 1),(NULL, "varicella zoster immune globulin (Investigational New Drug)", "VZIG (IND)", 117, 100, '', 0, 0, '', '', '', '', 1),(NULL, "human papilloma virus vaccine, bivalent", "HPV, bivalent", 118, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rotavirus, live, monovalent vaccine", "rotavirus, monovalent", 119, 100, '', 0, 0, '', '', '', '', 1),(NULL, "diphtheria, tetanus toxoids and acellular pertussis vaccine, Haemophilus influenzae type b conjugate, and poliovirus vaccine, inactivated (DTaP-Hib-IPV)", "DTaP-Hib-IPV", 120, 100, '', 0, 0, '', '', '', '', 1),(NULL, "zoster vaccine, live", "zoster", 121, 100, '', 0, 0, '', '', '', '', 1),(NULL, "rotavirus vaccine, unspecified formulation", "rotavirus, unspecified formulation", 122, 100, '', 0, 0, '', '', '', '', 1),(NULL, "influenza virus vaccine, H5N1, A/Vietnam/1203/2004 (national stockpile)", "influenza, H5N1-1203", 123, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Novel Influenza-H1N1-09, live virus for nasal administration", "Novel Influenza-H1N1-09, nasal", 125, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Novel influenza-H1N1-09, preservative-free, injectable", "Novel influenza-H1N1-09, preservative-free", 126, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Novel influenza-H1N1-09, injectable", "Novel influenza-H1N1-09", 127, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Novel influenza-H1N1-09, all formulations", "Novel Influenza-H1N1-09, all formulations", 128, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Japanese Encephalitis vaccine, unspecified formulation", "Japanese Encephalitis, unspecified formulation", 129, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Diphtheria, tetanus toxoids and acellular pertussis vaccine, and poliovirus vaccine, inactivated", "DTaP-IPV", 130, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Historical record of a typhus vaccination", "typhus, historical", 131, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Historical record of vaccine containing, diphtheria, tetanus toxoids and acellular pertussis, poliovirus, inactivated, Haemophilus influenzae type b conjugate, Hepatitis B (DTaP-Hib-IPV)", "DTaP-IPV-HIB-HEP B, historical", 132, 100, '', 0, 0, '', '', '', '', 1),(NULL, "pneumococcal conjugate vaccine, 13 valent", "Pneumococcal conjugate PCV 13", 133, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Japanese Encephalitis vaccine for intramuscular administration", "Japanese Encephalitis IM", 134, 100, '', 0, 0, '', '', '', '', 1),(NULL, "influenza, high dose seasonal, preservative-free", "Influenza, high dose seasonal", 135, 100, '', 0, 0, '', '', '', '', 1),(NULL, "meningococcal oligosaccharide (groups A, C, Y and W-135) diphtheria toxoid conjugate vaccine (MCV4O)", "Meningococcal MCV4O", 136, 100, '', 0, 0, '', '', '', '', 1),(NULL, "HPV, unspecified formulation", "HPV, unspecified formulation", 137, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus and diphtheria toxoids, not adsorbed, for adult use", "Td (adult)", 138, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Td(adult) unspecified formulation", "Td(adult) unspecified formulation", 139, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Influenza, seasonal, injectable, preservative free", "Influenza, seasonal, injectable, preservative free", 140, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Influenza, seasonal, injectable", "Influenza, seasonal, injectable", 141, 100, '', 0, 0, '', '', '', '', 1),(NULL, "tetanus toxoid, not adsorbed", "tetanus toxoid, not adsorbed", 142, 100, '', 0, 0, '', '', '', '', 1),(NULL, "Adenovirus, type 4 and type 7, live, oral", "Adenovirus types 4 and 7", 143, 100, '', 0, 0, '', '', '', '', 1),(NULL, "no vaccine administered", "no vaccine administered", 998, 100, '', 0, 0, '', '', '', '', 1),(NULL, "unknown vaccine or immune globulin", "unknown", 999, 100, '', 0, 0, '', '', '', '', 1)' due to: Incorrect double value: '' for column `openemr`.`codes`.`cyp_factor` at row 1```
### To Reproduce

1. Change SQL mode in the `Installer.class.php`
2. Run GUI installation (https://github.com/openemr/openemr/issues/3207)
3. See error

### Expected behavior
1. Runs without any error

### Client configuration

- Browser: Chrome
- Operating system: Mac OS X

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
Fixes #3365 #3367 

#### Short description of what this resolves:
We fix two issues here, one is related to remove blank insertions in the column `cyp_factor`, second one is small size of the column `code_text_short`

#### Changes proposed in this pull request:

- Removed blank insertions for the column `cyp_factor`
- Increased the size of varchar from 24 to 255 for the column `code_text_short`
-

### CODE REVIEW

File: sql/5_0_2-to-5_0_3_upgrade.sql [hunk 1]
```diff
@@ -434,4 +434,8 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 
 #IfNotColumnType form_eye_mag_prefs ordering smallint(6)
 ALTER TABLE `form_eye_mag_prefs` MODIFY `ordering` smallint(6) DEFAULT NULL;
+#EndIf
+
+#IfNotColumnType codes varchar(24)
```
Comment (bradymiller) [inline sql/5_0_2-to-5_0_3_upgrade.sql:439]:
```
#IfNotColumnType codes code_text_short varchar(255)
```

File: sql/database.sql [hunk 2]
```diff
@@ -1028,7 +1028,7 @@ DROP TABLE IF EXISTS `codes`;
 CREATE TABLE `codes` (
   `id` int(11) NOT NULL auto_increment,
   `code_text` varchar(255) NOT NULL default '',
-  `code_text_short` varchar(24) NOT NULL default '',
+  `code_text_short` varchar(50) NOT NULL default '',
```
Comment (bradymiller) [inline sql/database.sql:1031]: should be 255

**Comment by bradymiller** [comment]
hi @ShubhamBansal1997 , Noted a couple issues. Once those are fixed, will be ready for codebase. thanks! -brady

File: sql/database.sql (hunk 2 above)
Comment (ShubhamBansal1997) [reply sql/database.sql]: Fixed!!

File: sql/5_0_2-to-5_0_3_upgrade.sql (hunk 1 above)
Comment (ShubhamBansal1997) [reply sql/5_0_2-to-5_0_3_upgrade.sql]: Fixed!!

**Comment by ShubhamBansal1997** [comment]
Fixed the issues, feel free for a second review
cc @bradymiller

**Comment by bradymiller** [comment]
hi @ShubhamBansal1997 , Looks good and bringing into codebase. Thanks for the fixes! -brady

**Pull request merged**